docs/restic-with-minio.md

```diff
@@ -1,42 +1,58 @@
 # restic with Minio Server [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-`restic` is a fast, efficient and secure backup tool. It is an open source project available under ``BSD 2-Clause License``.
+`restic` is an open source backup tool that is fast, efficient, and secure. `restic` is available under the [BSD 2-Clause License](https://opensource.org/licenses/BSD-2-Clause).
 
-In this recipe we will learn how to use `restic` to backup data into Minio Server.
+This guide describes how to use `restic` to backup your data onto Minio Server. These are the steps you will follow:
 
-## 1. Prerequisites
+1. [Install Minio Server](#installserver) 
+2. [Install `restic`](#installrestic) 
+3. [Prepare the Environment for `restic`](#prepenv) 
+4. [Start Restic](#startrestic) 
+5. [Perform a Backup](#performbackup)
 
-Install Minio Server from [here](http://docs.minio.io/docs/minio-quickstart-guide).
+## <a name="installserver"></a>1. Install Minio Server
 
-## 2. Installation
+Install Minio Server using the instructions in the [Minio Quickstart Guide](http://docs.minio.io/docs/minio-quickstart-guide).
 
-Install restic from [https://restic.github.io](https://restic.github.io).
+## <a name="installrestic"></a>2. Install `restic`
 
-## 3. Configuration
+Install `restic` using these instructions: [https://restic.github.io](https://restic.github.io).
 
-Set Minio credentials in the environment variables as shown below.
+## <a name="prepenv"></a>3. Prepare the Environment for `restic`
+
+Store the access key and secret displayed by Minio Server in the respective environment variables:
 
 ```sh
 export AWS_ACCESS_KEY_ID=<YOUR-ACCESS-KEY-ID>
-export AWS_SECRET_ACCESS_KEY= <YOUR-SECRET-ACCESS-KEY>
+export AWS_SECRET_ACCESS_KEY=<YOUR-SECRET-ACCESS-KEY>
 ```
 
-## 4. Commands
+## <a name="startrestic"></a>4. Start Restic
 
-Start `restic` and point it to the bucket where the backup data will reside.
+As you start `restic`, specify the bucket where the backup data will reside:
 
 ```sh
 ./restic -r s3:http://localhost:9000/resticbucket init
 ```
 
-Copy backups from the local machine to the bucket on Minio server.  
+**Note:** When prompted, enter a password to assign to the repository.
+
+## <a name="performbackup"></a>5. Perform a Backup
+Use `restic` to create a backup copy of a directory on your local machine to a bucket on Minio Server:
 
 ```sh
 ./restic -r s3:http://localhost:9000/resticbucket backup /home/minio/workdir/Docs/
+```
+
+You should see a response similar to this one:
+
+```sh
 enter password for repository:
```

Comment:
The display on this is doing a strange thing, the words "for" and "in" are being highlighted in red, is this due to the inclusion of `sh`?

Reply:
@eco-minio Made changes per slack conversation, removed sh from ``` since this is just a response. That fixes the weird coloring.